### PR TITLE
Add asset-compute-pipeline support

### PR DESCRIPTION
**Pull request by pheenomenon**
Add capabilities to use the [asset-compute-pipeline](https://github.com/adobe/asset-compute-pipeline)

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

Fixes # (issue)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,9 +13,10 @@
   "dependencies": {
     "@adobe/asset-compute-commons": "^1.1.2",
     "@adobe/cgroup-metrics": "^3.0.3",
-    "@adobe/httptransfer": "^2.2.0",
+    "@adobe/httptransfer": "^2.7.0",
     "@adobe/metrics-sampler": "^1.0.2",
     "@adobe/node-fetch-retry": "^1.1.1",
+    "@nui/asset-compute-pipeline": "^0.1.4",
```
Comment (pheenomenon) [inline package.json:19]: Once asset-compute-pipeline is open sourced sdk will need updates in package.json.

File: package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "@adobe/asset-compute-sdk",
+  "name": "@nui/asset-compute-sdk",
   "description": "Adobe Asset Compute Worker SDK",
   "license": "Apache-2.0",
-  "version": "2.10.7",
+  "version": "0.0.1-pipeline-alpha",
```
Comment (pheenomenon) [inline package.json:5]: Once asset-compute-pipeline is open sourced sdk will need updates in package.json.

File: lib/api.js [hunk 3]
```diff
@@ -15,25 +15,45 @@
...
+ * One worker instantiates a pipeline, which executes one or more transformers in sequence
  *
  * @param {*} renditionCallback callback called for each rendition, must convert source into rendition
  *                              signature: (source, rendition)
  *                              required
  * @param {*} options optional options
  */
-function worker(renditionCallback, options={}) {
+function worker(renditionCallback, options = {}) {
+    console.log("worker(renditionCallback, options={})");
```
Comment (tmathern) [inline lib/api.js:31]: You can remove this log.

File: test-worker/worker.js [hunk 4]
```diff
@@ -23,8 +23,11 @@ const SUPPORTED_FMT = ["png", "jpg"];
 
 process.env.OPENWHISK_NEWRELIC_DISABLE_METRICS = true;
 process.env.SDK_POST_PROCESSING_TEST_MODE = true;
+process.env.DEBUG = "*";
 
 exports.main = worker(async (source, rendition) => {
+    console.log("In test-worker rendition callback", rendition)
```
Comment (tmathern) [inline test-worker/worker.js:29]: I think you can remove that log line too.

File: test/api.test.js [hunk 5]
```diff
@@ -788,6 +788,15 @@ describe("api.js", () => {
             await testUtil.assertSimpleParamsMetrics(receivedMetrics);
         });
 
+        it.skip('pipeline: should not return transformers if no manifest file found', async () => {
```
Comment (tmathern) [inline test/api.test.js:791]: Why keep the test if it's skipped?

**Review by tmathern — CHANGES_REQUESTED** [review]

**Review by tmathern — APPROVED** [review]

**Review by jdelbick — APPROVED** [review]

**Pull request merged**